### PR TITLE
feat(quality-slider): expose video constraint method and data channel…

### PR DESCRIPTION
**Pull request by virtuacoplenny**
… opening

### CODE REVIEW

File: JitsiConferenceEventManager.js [hunk 1]
```diff
@@ -437,6 +437,8 @@ JitsiConferenceEventManager.prototype.setupRTCListeners = function() {
         conference.room.connectionTimes['data.channel.opened'] = now;
         Statistics.analytics.sendEvent('conference.dataChannel.open',
             { value: now });
+
+        conference.eventEmitter.emit(JitsiConferenceEvents.DATA_CHANNEL_OPEN);
```
Comment (bbaldino) [inline JitsiConferenceEventManager.js]: is there already an event for the websocket version?

Comment (virtuacoplenny) [inline JitsiConferenceEventManager.js]: They should go through the same abstraction. Both peerconnection and websocket go through BridgeChannel's _handleChannel(), which sets the channel.onopen callback, which should fire RTCEvents.DATA_CHANNEL_OPEN, which conference event manager reacts to.

Comment (bbaldino) [inline JitsiConferenceEventManager.js]: ah that's a bummer...this event should've been renamed then it seems like.

File: JitsiConferenceEvents.js [hunk 2]
```diff
@@ -57,6 +57,11 @@ export const CONNECTION_INTERRUPTED = 'conference.connectionInterrupted';
 export const CONNECTION_RESTORED = 'conference.connectionRestored';
 
 /**
+ * A connection to the video bridge's data channel has been established.
+ */
+export const DATA_CHANNEL_OPEN = 'conference.dataChannelOpen';
```
Comment (yanas) [inline JitsiConferenceEvents.js]: We can rename this to DATA_CHANNEL_OPENED for consistency.

Comment (virtuacoplenny) [inline JitsiConferenceEvents.js]: That would make it inconsistent with RTCEvents.DATA_CHANNEL_OPEN. I'm not opposed to changing it, though.

Comment (yanas) [inline JitsiConferenceEvents.js]: I see. I still would prefer to make it consistent with the rest of the events in this class. Even though it's unfortunate we haven't followed the consistency in RTCEvents.

**Pull request merged**